src/components/Frontend.tsx

```diff
@@ -142,101 +142,187 @@ export default function Frontend(
+                    onClick={() => {
+                      reportSybil({
+                        reporterFid: BigInt(reporterFid || 0),
+                        targetFid: BigInt(targetData?.fid || 0),
+                        reportedAsSybil: false,
+                      });
+                    }}
+                    disabled={!isConnected || !isVerified}
+                  >
+                    Report Human
+                  </Button>
+                  <Button
+                    className="w-full h-12 bg-red-700 hover:bg-red-600 text-white disabled:bg-gray-500"
+                    onClick={() => {
+                      reportSybil({
+                        reporterFid: BigInt(reporterFid || 0),
+                        targetFid: BigInt(targetData?.fid || 0),
+                        reportedAsSybil: true,
+                      });
+                    }}
+                    disabled={!isConnected || !isVerified}
+                  >
+                    Report Sybil
+                  </Button>
+                </div>
+              </>
+            )}
+            <div className="w-full flex flex-col">
+              <Button
+                className="w-full bg-red-700 hover:bg-red-600 disabled:bg-gray-500"
+                onClick={addFrame}
+                disabled={added}
+              >
+                Add frame to client
+              </Button>
+              <span>{addFrameResult}</span>
+            </div>
+
+            <Button
+              className="w-full bg-red-700 hover:bg-red-600"
+              onClick={() =>
+                isConnected
+                  ? disconnect()
+                  : connect({ connector: config.connectors[0] })
+              }
+            >
+              {isConnected ? "Disconnect wallet" : "Connect wallet"}
+            </Button>
+          </>
+        )}
+
+        {success === true && (
+          <>
+            <div className="w-full h-full flex flex-col gap-4 items-center justify-start">
+              <span>Thanks for your report!</span>
+              <Button
+                className="w-full h-12 bg-red-700 hover:bg-red-600 text-white"
+                onClick={() => {
+                  sdk.actions.close();
+                }}
+              >
+                Close frame
+              </Button>
+            </div>
+          </>
+        )}
+
+        {success === false && (
+          <>
+            <div className="w-full h-full flex flex-col gap-4 items-center justify-start">
+              <span>Sorry, an unexpected error occurred!</span>
+              <Button
+                className="w-full h-12 bg-red-700 hover:bg-red-600 text-white"
+                onClick={() => {
+                  sdk.actions.close();
+                }}
+              >
+                Close frame
+              </Button>
+            </div>
+          </>
+        )}
```

Comment:
I think that you mentioned this yesterday, but it would be good to break it into components